rosidl_typesupport_introspection_cpp/package.xml

```diff
@@ -14,6 +14,8 @@
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
   <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
 
+  <exec_depend>rosidl_parser</exec_depend>
```

Comment:
need rosidl_cmake? It's added as an import in rosidl_typesupport_introspection_cpp/rosidl_typesupport_introspection_cpp/__init__.py